src/currentprocess/filesource.rs

```diff
@@ -1,12 +1,9 @@
 use std::io::{self, BufRead, Read, Result, Write};
-#[cfg(feature = "test")]
-use std::{
-    io::Cursor,
-    sync::{Arc, Mutex, MutexGuard},
-};
 
 use enum_dispatch::enum_dispatch;
 
+#[cfg(feature = "test")]
+pub(crate) use self::tests::*;
```

Comment:
I would put this near the `tests` module definition (probably underneath).

Reply:
Done.